src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Bulldozer(), new Truck(), new Excavator()};
+        printResult(machines);
+    }
 
+    public static void printResult(Machine[] machines) {
+        for (Machine working : machines) {
```

Comment:
Why did you named this variable in for-each loop `working`? It's better to name it `machine`.